tools/protoc/protoc

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
As of right now this only supports OSX, but this is only temporary, I'm working on a generalized solution